tests/tests.py

```diff
@@ -16,6 +17,10 @@
 from django_keyerror.app_settings import app_settings
 
 
+if sys.version_info >= (3, 0, 0):
```

Comment:
Prefrably `six.PY3` as this makes it easier to migrate away Python 2.x support later simply by grepping for `six` ;)

Reply:
D'oh, of course. 40a1682d0850a191ccbdfc556b910fcb479bbffd